src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -218,4 +218,13 @@ In order to debug this project, add an executable project to this solution which
   <data name="Restore_EmptyTargetFrameworkMoniker" xml:space="preserve">
     <value>The value of the 'TargetFrameworkMoniker' and 'NuGetTargetMoniker' properties in the '{0}' configuration are both empty. This configuration will not contribute to NuGet restore, which may result in restore and build errors.</value>
   </data>
+  <data name="WindowsFormEditor_DisplayName" xml:space="preserve">
```

Comment:
pardon my ignorance, but why are the old names not removed?

Reply:
Do you mean from legacy?

Reply:
yes, are they still used?

Reply:
Did you have "old" names in CPS before your change?

Reply:
CPS lives in a different code base. That will be deleted from there.